dm/config/subtask.go

```diff
@@ -291,6 +293,8 @@ type SubTaskConfig struct {
 	ExtStorage      extstorage.ExternalStorage `toml:"-" json:"-"`
 	MetricsFactory  promutil.Factory           `toml:"-" json:"-"`
 	FrameworkLogger *zap.Logger                `toml:"-" json:"-"`
+	UUID            string                     `toml:"-" json:"-"`
```

Comment:
Could you add some comments about how does it work.

Reply:
Do you have a better name @GMHDBJD I'll rename it and add some comments in next commit